packages/wrangler/src/__tests__/guess-worker-format.test.ts

```diff
@@ -65,4 +67,17 @@ describe("guess worker format", () => {
     );
     expect(guess).toBe("service-worker");
   });
+
+  it("logs a warning when a worker has exports, but not a default one", async () => {
+    await writeFile("./index.ts", "export const foo = 1;");
+    const guess = await guessWorkerFormat(
+      path.join(process.cwd(), "./index.ts"),
+      process.cwd(),
+      undefined
+    );
+    expect(guess).toBe("service-worker");
+    expect(std.warn).toMatchInlineSnapshot(
+      `"The entrypoint index.ts has exports like an ES Module, but hasn't defined a default export like a module worker normally would. Assuming that it's a service-worker format worker."`
```

Comment:
"Assuming" is bad phrasing here. Ideas for a better sentence?

Reply:
`Building the Worker using "service-worker" format`??

Reply:
I like that better, thanks!